shipyard_blueprints/bigquery/shipyard_bigquery/shipyard/download.py

```diff
@@ -0,0 +1,69 @@
+import os
+import argparse
+import sys
+import shipyard_bp_utils as shipyard
+
+
+from shipyard_templates import ShipyardLogger, ExitCodeException
+from shipyard_bigquery import BigQueryClient
+
+
+logger = ShipyardLogger.get_logger()
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--query", dest="query", required=True)
+    parser.add_argument("--service-account", dest="service_account", required=True)
+    parser.add_argument(
+        "--destination-file-name",
+        dest="destination_file_name",
+        default="output.csv",
+        required=True,
+    )
+    parser.add_argument(
+        "--destination-folder-name",
+        dest="destination_folder_name",
+        default="",
+        required=False,
+    )
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    try:
+        args = get_args()
+        target_folder = (
+            args.destination_folder_name if args.destination_folder_name != "" else None
```

Comment:
can be simplified to 
```
if target_folder := args.destination_folder_name:
...
```

Reply:
going to leave the walrus operator out of it this time